### ISSUE TITLE
How can I use ChartJS-Node-Canvas in react or front end application?

### ISSUE THREAD

**Issue by DipaliShah1193** [SeanSobey/ChartjsNodeCanvas#99]

**Comment by typpo** [comment]
For a frontend application, I would suggest constructing the chart normally with Chart.js and using the built-in [toBase64Image](https://www.chartjs.org/docs/latest/developers/api.html#tobase64image-type-quality) function to get a PNG or JPEG image of the chart.

**Comment by SeanSobey** [comment]
Yes, this library is a kit specifically for running Chartjs on Nodejs. Chartjs is a front end based library so you can use it as is or a wrapper like [this](https://github.com/reactchartjs/react-chartjs-2) for easy integration into react (based on a quick search, I have not used this).

Only if you looking to render the charts server-side for some specific reason, or are maybe doing server-side rendering of pages should you look to use this lib.
